src/main/java/seedu/duke/BusStops.java

```diff
@@ -12,7 +12,7 @@
     MUSEUM("Museum", "Opposite Faculty of Engineering"),
     UNIVERSITYTOWN("University Town","place full of fun"),
     UNIVERSITYHEALTHCENTRE("University Health Centre","UHC, where students can get MC"),
-    OPPKENTRIDGEMRTSTATION("OppKentRidgeMRTstation","MRT"),
+    OPPKENTRIDGEMRTSTATION("Opp Kent Ridge MRT station","MRT"),
     RAFFLESHALL("Raffles Hall","Hall located nearby Faculty of Engineering"),
     KENTVALE("Kent Vale","somewhere"),
```

Comment:
```suggestion
    KENTVALE("Kent Vale","Accommodation for visiting faculty guests"),
```

Reply:
Hahah, okie! I will push these changes in a new commit!